src/actions/__tests__/historyActions.test.js

```diff
@@ -349,9 +349,25 @@ describe('History Actions', () => {
         });
       });
 
-      it('should handle the wrong eth value that api could return', () => {
+      it('should handle the wrong 1e-22 eth value that api could return', () => {
         expect(parseEthValue(1e-22)).toBe('100000000000000');
       });
+
+      it('should handle the 0.00104898 eth value that api could return', () => {
+        expect(parseEthValue(0.00104898)).toBe('1048979999999999');
```

Comment:
Shouldn't the `0.00104898` be equal to `1048980000000000`?

Reply:
haha, good catch, overthought something, fixed! 👍